docs/references/general/fitacf.md

```diff
@@ -1,2 +1,175 @@
+| *ercod*                 | **None**         | ***short***  | Error Code                                                                              |
+| *stat.agc*              | **None**         | ***short***  | AGC status word                                                                         |
+| *stat.lopwr*            | **None**         | ***short***  | LOPWR status word                                                                       |
+| *noise.search*          | **None**         | ***float***  | Calculated noise from clear frequency search                                            |
+| *noise.mean*            | **None**         | ***float***  | Average noise from clear frequency band                                                 |
+| *channel*               | **None**         | ***short***  | Channel number for variant changes in radars parameters in one scan, ex) frequency      |
+| *bmnum*                 | **None**         | ***short***  | Beam number                                                                             |
+| *bmazm*                 | **None**         | ***float***  | Beam azimuth                                                                            |
+| *scan*                  | **None**         | ***short***  | Scan flag, switches between 0 to 1 or 0 to -1 to indicate a new scan                    |
+| *offset*                |  *us* | ***short***  | Offset between channels for a stereo radar                                                       |
+| *rxrise*                | *us* | ***short***  | Receiver rise time                                                                      |
+| *intt.sc*               | *s*        | ***short***  | whole number of seconds of integration time                                             |
+| *intt.us*               | *us* | ***int***    | Fractional number of microseconds of integration time                                    |
+| *txpl*                  | *us* | ***short***  | Transmitted pulse length                                                                |
+| *mpinc*                 | *us* | ***short***  | Multi-pulse increment                                                                   |
+| *mppul*                 | **None**         | ***short***  | Number of pulses in the sequence                                                        |
+| *mplgs*                 | **None**         | ***short***  | Number of lags in the sequence                                                          |
+| *mplgexs*              | **None**         | ***short***  |   Number of lags used for tauscan. This field originates from `fittex` algorithm and is not used in FITACF file format. | 
+| *ifmode*                | **None**         | ***short***  | Receiver was processing at an IF (intermediate frequency) instead of RF (radio frequency) band       |
```

Comment:
Just wanted to check before suggesting an edit. Voltage samples are collected by the receiver, either by:

1. mixing to baseband (see heterodyne or superheterodyne receivers) and then digitizing the signal, or
2. mixing to an IF and digitizing

With Borealis and the UAF SuperDARN radars, they can directly digitize the received signal without the need for mixing. So here's my suggested edits:


```suggestion
| *ifmode*                | **None**         | ***short***  |  A flag indicating that the receiver voltage samples were obtained by digitizing the signal at an intermediate frequency (and then digitally mixed to baseband) instead of mixing to baseband and then digitizing (`ifmode=0`).         |
```

If this PR takes weeks, then sure, maybe we're being too pedantic. It's only been open for 8 days? It would be nice to have descriptions that are useful. When I did my PhD, I had to suffer through the existing descriptions which in some cases are too short to be meaningful and/or cryptic.